content/inferno/time-series/time-series.md

```diff
@@ -0,0 +1,309 @@
+
+## Time Series Classification
+
+The problem of time series classification is of _primary importance_ in the world of data mining, and over the last two decades, countless methods have been proposed to solve it. 
+
+Knowing how to predict the best choices in the near future is even vital in a variety of industrial scenarios or even _critical_ scenarios such as aircraft safety systems!
+
+Some examples of time series classification problems are:
+
+- Predict whether a machine might break or not
+- predict whether a customer will leave a service or not
+- classify the type of disease of a patient by the time series of his heartbeat
+- classify an animal according to the sound it makes
+- predict anomalies and trend changes in quantities measured by the sensors 
+- and many more...
+
+In this section of the guide, you can find a logical collection of all the resources that can be useful to take advantage of **the power of modern time series classification methods**.
+
+First of all, the site you must ** refer to** to find all the latest _datasets, techniques, papers, and code_ that you can use to solve your time series classification problems:
+
+> This website is an ongoing project to develop a comprehensive repository for research into time series classification. 
+
+- [**Time Series Classification Website**](http://www.timeseriesclassification.com/)
+
+For a complete comparison of all available useful methods and their tradeoffs, read:
+
+- [The Great Time Series Classification Bakeoff: An Experimental Evaluation](https://www.researchgate.net/profile/Anthony_Bagnall/publication/301856632_The_Great_Time_Series_Classification_Bake_Off_An_Experimental_Evaluation_of_Recently_Proposed_Algorithms_Extended_Version/links/579b580e08ae80bf6ea33d12.pdf)
+- [Deep learning for time series classification: a review](https://arxiv.org/pdf/1809.04356.pdf)
+
+A very hot (and very recent) method that has been proposed and you should be aware of:
+
+- [ROCKET: Exceptionally fast and accurate time series classification using random convolutional kernels](https://arxiv.org/abs/1910.13051)
+
+Another extremely effective method that is always worth trying is that of the [matrix profile](#The-Matrix-Profile) (next section).
+
+::: tip
+Some methods are more precise, others more computationally performing, others can find complex relationships in the data (neural networks). 
+**A fundamental point** is that _rarely_ it is worth choosing complex methods (such as a complicated recurrent neural network) over more "simple" ones, such as BOSS or DTW.
+:::
+
+Equipped with the methods considered here, it's almost guaranteed that you will find the perfect fit for the requirements of your problems.
+
+## The Matrix Profile
+
+From the [The UCR Matrix Profile Page](https://www.cs.ucr.edu/~eamonn/MatrixProfile.html) website:
```

Comment:
Did you know of these:
- https://www.cs.ucr.edu/~eamonn/Matrix_Profile_Tutorial_Part1.pdf
- https://www.cs.ucr.edu/~eamonn/Matrix_Profile_Tutorial_Part2.pdf

Reply:
yes , the first one is already present, and I though that the second one is less important cause it explain the math reasons behind the matrix profile approach... but it can be useful to post it